src/main/groovy/life/qbic/portal/sampletracking/web/views/ControlElements.groovy

```diff
@@ -17,7 +17,7 @@ import java.time.LocalDateTime
 @Log4j2
 class ControlElements extends GridLayout {
 
-    final static List<Status> SAMPLE_STATUSES = [Status.SAMPLE_QC_PASS, Status.SAMPLE_QC_FAIL, Status.SEQUENCING, Status.SEQUENCING_COMPLETE]
+    final static List<Status> SAMPLE_STATUSES = [Status.SAMPLE_RECEIVED, Status.SAMPLE_QC_PASS, Status.SAMPLE_QC_FAIL, Status.LIBRARY_PREP_FINISHED]
```

Comment:
Just have seen it now: why don't we have the past term for the adjectives consistently? 
```suggestion
    final static List<Status> SAMPLE_STATUSES = [Status.SAMPLE_RECEIVED, Status.SAMPLE_QC_PASSED, Status.SAMPLE_QC_FAILED, Status.LIBRARY_PREP_FINISHED]
```

Reply:
We cannot do the PASSED and FAILED now because we would have to deprecate here. `DATA_AT_QBIC` would also not fit.